galleries/examples/images_contours_and_fields/colormap_normalizations.py

```diff
@@ -13,85 +13,86 @@
 
 import matplotlib.colors as colors
 
-# %%
-# Lognorm: Instead of pcolor log10(Z1) you can have colorbars that have
-# the exponential labels using a norm.
-
 N = 100
-X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
 
-# A low hump with a spike coming out of the top.  Needs to have
-# z/colour axis on a log scale, so we see both hump and spike.
-# A linear scale only shows the spike.
+# %%
+# LogNorm
+# -------
+# This example data has a low hump with a spike coming out of its center. To see both
+# hump and spike, this requires the z/colour axis on a log scale; a linear scale only
+# shows the spike.
+#
+# Instead of transforming the data directly with ``pcolor(log10(Z))``, the colorbars can
+# have exponential labels using a `.LogNorm`.
 
+X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
 Z1 = np.exp(-X**2 - Y**2)
 Z2 = np.exp(-(X * 10)**2 - (Y * 10)**2)
 Z = Z1 + 50 * Z2
 
 fig, ax = plt.subplots(2, 1)
 
-pcm = ax[0].pcolor(X, Y, Z,
-                   norm=colors.LogNorm(vmin=Z.min(), vmax=Z.max()),
-                   cmap='PuBu_r', shading='nearest')
-fig.colorbar(pcm, ax=ax[0], extend='max')
+pcm = ax[0].pcolor(X, Y, Z, cmap='PuBu_r', shading='nearest',
+                   norm=colors.LogNorm(vmin=Z.min(), vmax=Z.max()))
+fig.colorbar(pcm, ax=ax[0], extend='max', label='LogNorm')
 
 pcm = ax[1].pcolor(X, Y, Z, cmap='PuBu_r', shading='nearest')
-fig.colorbar(pcm, ax=ax[1], extend='max')
+fig.colorbar(pcm, ax=ax[1], extend='max', label='LinearNorm')
```

Comment:
```suggestion
fig.colorbar(pcm, ax=ax[1], extend='max', label='linear scaling')
```
Or similar. unfortunately, the linear scaling is named `Normalize` and not `LinearNorm`, so using camel case `LinearNorm` is somewhat misleading. also in the following examples.